lib/traject/macros/custom.rb

```diff
@@ -132,7 +132,7 @@ def sought_link_data_exists?(link_type, field)
       end
 
       def fulltext_link_available?(ind1, ind2, url_label)
-        (ind2 == '0' || ind1 == '4') && !NOT_FULLTEXT.match?(url_label)
+        (ind2 == '0' || ind1 == '4' || (ind1.strip.empty? && ind2.strip.empty?)) && !NOT_FULLTEXT.match?(url_label)
```

Comment:
I'm not sure I see why `strip` is needed. Looking at the spec it looks like an empty string is passed... 🤔

Reply:
yeah this was interesting to me too, the example record I was testing came with whitespace in its indicators like `" "` so I had to add that strip 🤔

Reply:
ah ok, i was just going by the spec. Did you also see cases where the string was just a true empty string (i.e., `""`)?

Reply:
I tried the example records we have in the issue and they all had whitespace in both indicators. of course I am not sure if that is the case always.

```
1174303
23786738
2206901
2187375
```

Reply:
Ah I was just thrown off by the spec which passes empty string not `" "`

Reply:
yeah, I will make the specs like this `" "`

Reply:
🙏 danke